### ISSUE TITLE
Ternary operator messes with variable member highlighting

### ISSUE THREAD

**Issue by jeff-hykin** [jeff-hykin/cpp-textmate-grammar#108]
From https://github.com/Microsoft/vscode-cpptools/issues/3006
```
cfg.t_low_fan ? (cfg.t_low_fan == 1 ? "low" : "high") : "off";
```

**Comment by matter123** [comment]
This may be already fixed (with 1.8.4)

C | C++
---|---
![Screenshot from 2019-04-17 21-53-38](https://user-images.githubusercontent.com/714007/56337707-6d965e00-615b-11e9-8020-8b12ed944ebe.png) | ![Screenshot from 2019-04-17 21-53-50](https://user-images.githubusercontent.com/714007/56337711-7424d580-615b-11e9-8312-806a39876063.png)

**Comment by jeff-hykin** [comment]
<img width="743" alt="Screen Shot 2019-04-18 at 12 06 43 AM" src="https://user-images.githubusercontent.com/17692058/56338039-e43c5700-616d-11e9-9ba6-4b6960ad47f5.png">
Theres what it looks like for me on `v1.8.4`
